composer.json

```diff
@@ -20,6 +19,7 @@
     "phpunit/phpunit": "^5.7 || ^6.5 || ^7.5",
     "roave/security-advisories": "dev-master",
     "sirbrillig/phpcs-variable-analysis": "^2.8",
+    "szepeviktor/phpstan-wordpress": "^0.6.5",
```

Comment:
We require PHP 5.6, so that's not gonna work

Reply:
Forget it....
See next commit